test/TestUINodes/TestUINodes.csproj

```diff
@@ -31,49 +28,60 @@
   </ItemGroup>
   <!--net6.0 net8.0 etc-->
   <ItemGroup Condition=" '$(TargetFramework)' != 'net48' ">
-    <PackageReference Include="NUnit" Version="3.13.3" />
-    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+    <PackageReference Include="NUnit" Version="3.13.3">
+      <IncludeAssets>none</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1">
+      <IncludeAssets>none</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\DynamoCoreWpf\DynamoCoreWpf.csproj">
       <Project>{51bb6014-43f7-4f31-b8d3-e3c37ebedaf4}</Project>
       <Name>DynamoCoreWpf</Name>
       <Private>False</Private>
+      <IncludeAssets>none</IncludeAssets>
```

Comment:
This is to prevent these binaries from being copied into the `nodes` subfolder?

Reply:
yes, I tried using @pinzart90's suggestion of using 
```
<ItemDefinitionGroup>
    <ReferenceCopyLocalPaths>
	  <!--Copy all assembly references to the OutputPath parent dir (/nodes/../) -->
	  <DestinationSubDirectory>..\</DestinationSubDirectory>
    </ReferenceCopyLocalPaths>
  </ItemDefinitionGroup>
```

but I don't think this is the correct solution - and I will file a followup to do something more idiomatic for the other projects that use it.

For example, it does not work for this TestUINodes.csproj project - because this project references other binaries in the `nodes` folder - it then copies those binaries, not just their deps, to the `bin` folder - so we end up with things like `UnitsUI` in both `bin` and `nodes`.